zmq/src/jbuild

```diff
@@ -4,6 +4,7 @@
  ((name zmq)
   (public_name zmq)
   (wrapped false)
+  (flags (:standard "-w" "-37"))
```

Comment:
What triggers warning 37?

Reply:
Lots of (with different kind of constructors). 

```File "zmq/src/ZMQ.ml", line 663, characters 0-514:
Error (warning 37): constructor I_ETERM is never used to build values.
(However, this constructor appears in patterns.)
```
I don't think ocaml can see that these are constructed in the C code.

Reply:
Makes sense - thanks.